app/grandchallenge/algorithms/templates/algorithms/job_detail.html

```diff
@@ -80,11 +80,7 @@ <h2>Algorithm Result</h2>
                         {% for object in json %}
                             <tr>
                                 <td>{{ object.interface.title }}</td>
-                                {% if object.interface.kind == 'STR' %}
-                                    <td>{{ object.value|slice:"1:-1" }} </td>
-                                {% else %}
-                                    <td>{{ object.value }}</td>
-                                {% endif %}
+                                <td>{{ object.value }}</td>
```

Comment:
It seems to me that this was handling the json `"` character. Could there be a difference in how these data are stored? Did you try this with interfaces with values stored in the db and in the object store? I know that the contents of the file `text-report.json` will be `"My fancy report"`.

Reply:
Hm, I didn't know that we had string interfaces not stored in the DB, sorry! Strings stored as json files would currently not show at all (i.e. you would see `None`) because we print `civ.value`. Do we want to show the contents of those json files in the table though? We don't this for other json interfaces not stored in the db. They could potentially contain a lot of content, maybe a download button would be better then. We don't show the

Reply:
I agree, we shouldn't show them. I'm just a bit confused why the `"` was necessary in the first place now. But, if it's definitely not required then we can go ahead, let me know. We could of course do conditional removal (e.g. if the first and last char are `"` then remove them) with a template tag.

Reply:
Yes, I was trying to understand that as well. No idea to be honest. I can confirm that there is no need for removing `"`, but we do need to add some code to make sure that interfaces not stored in the DB don't get added to the display. I will commit those additions in a bit.

Reply:
OK lets keep an eye on it after we deploy